addons/bus/static/src/bus_parameters_service.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+export const busParametersService = {
+    start() {
+        return {
+            serverURL: window.origin,
+        };
+    },
+};
+
+registry.category("services").add("bus_parameters", busParametersService);
```

Comment:
Could name service `bus.parameters`, so it's clearly intended for `bus` module (rather misleading for broad "bus" terminology)